gmakefile.test

```diff
@@ -1,6 +1,7 @@
 # -*- mode: makefile-gmake -*-
 
-CONFIGDIR := config
+# Account for PETSc prefix installation (empty/unset PETSC_ARCH)
+PETSCCONFIGDIR = $(if $(PETSC_ARCH), $(PETSC_DIR)/config, $(PETSC_DIR)/share/petsc/examples/config)
```

Comment:
```suggestion
PETSCCONFIGDIR ?= $(if $(PETSC_ARCH), $(PETSC_DIR)/config, $(PETSC_DIR)/share/petsc/examples/config)
```
it is already set when gmakefile includes gmakefile.test.

For consistency, this change should also be in gmakefile.

Reply:
> For consistency, this change should also be in gmakefile.

Is the change in ef39a54 what you mean?

Otherwise ok?